src/js/components/Pagination/PaginationSummary.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react';
+import { Box } from '../Box';
+import { Text } from '../Text';
+import { MessageContext } from '../../contexts/MessageContext';
+
+export const PaginationSummary = ({ messages, numberItems, page, step }) => {
```

Comment:
would a `...rest` be valuable here?

Reply:
good question.. I didnt put `…rest` was because its just a prop and now no one can pass anything extra but I see having it for the future however would we want the `…rest` on the  Box or Text. I would think the outer Text bc it seems like you’d more likely want to be affected text props rather than box props,

Reply:
IHMO devs often want to add a data-testid, hack padding, alignment etc so I think the ...rest would be good to have and pass to the main Box. That still leaves us the option to add a separate param for internal tweaks on text, similar to what we do with say dropProps etc.

Reply:
☝️ that makes sense if we put `...rest` on the Box than do something like textProps for the Text. I forget sometimes teams depend on the `data-testid` (which isn't great but reality)